src/components/Footer.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useContext } from 'react';
+import classNames from 'classnames';
+import { TodoContext } from '../contexts/TodoContext';
+import { Todo } from '../types/Todo';
+import { FilterStatus } from '../types/Status';
+
+export const Footer: React.FC = () => {
+  const { todos, setStatus, status } = useContext(TodoContext);
+  const visibleTodo = todos.some(todo => todo.completed);
```

Comment:
if we have boolean value its better to start name with `is` or `has`
`hasVisibleTodo`